pandas/plotting/_misc.py

```diff
@@ -274,12 +274,13 @@ def andrews_curves(
 
     Andrews curves have the functional form:
 
-    f(t) = x_1/sqrt(2) + x_2 sin(t) + x_3 cos(t) +
-           x_4 sin(2t) + x_5 cos(2t) + ...
+    .. math::
```

Comment:
Can you show a screen shot how this renders?

Reply:
![image](https://user-images.githubusercontent.com/23830955/216832740-8c354a9b-25a1-48d3-9e90-3dccbcfe9751.png)